pages/common/tailscale-up.md

```diff
@@ -0,0 +1,36 @@
+# tailscale up
+
+> Connects the client to the Tailscale network.
+> More information: <https://tailscale.com/kb/admin/>.
+
+- Connect to Tailscale:
+
+`sudo tailscale up`
+
+- Connect and offer the current machine to be an exit node for the internet traffic:
+
+`sudo tailscale up --advertise-exit-node`
+
+- Connect using a specific node for the internet traffic:
+
+`sudo tailscale up --exit-node={{exit_node_ip}}`
+
+- Connect and block incoming connections to the current node:
+
+`sudo tailscale up --shields-up`
+
+- Connect and accept DNS configuration from the admin panel:
+
+`sudo tailscale up --accept-dns={{true|false}}`
```

Comment:
Not sure if we should include the default argument here.

Reply:
Then we should add ``(defaults to `true`)`` in the example description:
```suggestion
`sudo tailscale up --accept-dns={{false}}`
```

Reply:
But then, we should also change the description to

> Connect and don't accept DNS configuration from the admin panel: